python/mxnet/gluon/rnn/rnn_cell.py

```diff
@@ -630,7 +630,7 @@ def hybrid_forward(self, F, inputs, states, i2h_weight,
         return next_h, [next_h]
 
 
-class SequentialRNNCell(RecurrentCell):
+class SequentialRNNCell(HybridRecurrentCell):
```

Comment:
This might break existing code where the member cells are not HybridRecurrentCells. Maybe have a separate class like HybridSequentialRNNCell?